packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -81,15 +82,20 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
     const address = watch(inputName);
     const options = getDefaultValue('options', []);
     const broadcastEnabled = options.includes('broadcast');
-    const inputState = getInputState(addressError);
+    const inputState = () => {
+        if (addressError) {
+            return getInputState(addressError);
+        }
+
+        return undefined;
+    };
 
     const handleQrClick = useCallback(async () => {
         const uri = await dispatch(scanOrRequestSendFormThunk()).unwrap();
 
         if (typeof uri !== 'string') {
             return;
         }
-
```

Comment:
nit: I prefer having whitespecas to separate code, it imo makes it more readable